models/tokens/optimism/tokens_optimism_erc20.sql

```diff
@@ -181,12 +181,34 @@ SELECT LOWER(contract_address) AS contract_address, symbol, decimals
 ,('0xf390830DF829cf22c53c8840554B98eafC5dCBc2','anyUSDC',6)
 ,('0x965f84d915a9efa2dd81b653e3ae736555d945f4','anyWETH',18)
 ,('0x1ccca1ce62c62f7be95d4a67722a8fdbed6eecb4','Multichain alETH',18)
-,('0x922D641a426DcFFaeF11680e5358F34d97d112E1','anyFXS',18)
-,('0xa3A538EA5D5838dC32dde15946ccD74bDd5652fF','sINR', 18)
-,('0xEe9801669C6138E84bD50dEB500827b776777d28','O3', 18)
-,('0xb12c13e66AdE1F72f71834f2FC5082Db8C091358','ROOBEE', 18)
-,('0xC22885e06cd8507c5c74a948C59af853AEd1Ea5C','USDD', 18)
-,('0xd6909e9e702024eb93312B989ee46794c0fB1C9D','BICO', 18)
-,('0x62BB4fc73094c83B5e952C2180B23fA7054954c4','PTaOptUSDC', 6)
-  
+,('0x922d641a426dcffaef11680e5358f34d97d112e1','anyFXS',18)
+,('0xa3a538ea5d5838dc32dde15946ccd74bdd5652ff','sINR', 18)
+,('0xee9801669c6138e84bd50deb500827b776777d28','O3', 18)
+,('0xb12c13e66ade1f72f71834f2fc5082db8c091358','ROOBEE', 18)
+,('0xc22885e06cd8507c5c74a948c59af853aed1ea5c','USDD', 18)
+,('0xd6909e9e702024eb93312b989ee46794c0fb1c9d','BICO', 18)
+,('0x62bb4fc73094c83b5e952c2180b23fa7054954c4','PTaOptUSDC', 6)
+,('0x66e8617d1df7ab523a316a6c01d16aa5bed93681','SHACK', 18)
+,('0x00a35fd824c717879bf370e70ac6868b95870dfb','IB', 18)
+,('0xf899e3909b4492859d44260e1de41a9e663e70f5','RADIO', 18)
```

Comment:
Please remove one of these two lines to avoid duplicates @chuxinh

Reply:
done, thanks!

Reply:
thanks @chuxinh, lgtm !